moto/awslambda_simple/responses.py

```diff
@@ -0,0 +1,12 @@
+from moto.core.responses import BaseResponse
+from ..awslambda.responses import LambdaResponse
+from .models import lambda_simple_backends, LambdaBackend
+
+
+class LambdaSimpleResponse(BaseResponse):
+    def __init__(self) -> None:
+        super().__init__(service_name="awslambda_simple")
```

Comment:
@rafcio19 I haven't tested it, but this may be the cause of the error that you're seeing.

Moto uses this service-name to instantiate a `boto3`-client (in order to figure out which action the incoming request should be mapped to). So that means that boto3 has to understand the value that we use here.

https://github.com/getmoto/moto/blob/6323c6c255851228fbdc3501a8348043a4940fa7/moto/core/responses.py#L488